README.md

```diff
@@ -1,14 +1,27 @@
 # Edvart
 
-Exploratory Data Analysis (EDA) is the initial task a data scientist or data
-analyst undertakes when they obtain new data. EDA refers to the critical
-process of conducting preliminary investigations on data to uncover patterns,
-spot anomalies, test hypotheses, and verify assumptions with the help of
-summary statistics and graphical representations.
+Edvart is an open-source Python library designed to simplify and streamline
+your exploratory data analysis (EDA) process.
+Edvart supports different levels of customization:
+from a default report generated in one line of code to a fully-customized
+report down to the level of code generating the visualizations.
```

Comment:
I think that this sentence is hard to comprehend without prior knowledge of Edvart. I think the intro would be better without it. I.e. just a quick intro line (Edvart is an open-source...) and then right away key features.

Reply:
Good point. The information in this sentence is repeated in the key features anyway.